@types/pal/image.d.ts

```diff
@@ -0,0 +1,30 @@
+declare module 'pal/image' {
+    type ImageSource = import('pal/image/types').ImageSource;
+    type IMemoryImageSource = import('pal/image/types').IMemoryImageSource;
+    export class ImageData {
+        constructor (imageAsset?: ImageSource | ArrayBufferView);
+        //constructor (content: string, manifestRoot: string);
+        destroy (): void;
+        get src (): string;
+        set src(url: string);
+        set data(value: any);
+        get data(): ImageSource | ArrayBufferView;
+        set width(value: any);
+        get width(): number;
+        get height(): number;
+        set height(value: any);
+        get format(): number;
+        get compressed(): number;
```

Comment:
```suggestion
        get compressed(): boolean;
```

this seems should return a boolean